src/boot/efi/stub.c

```diff
@@ -25,6 +26,8 @@ DECLARE_NOALLOC_SECTION(".sdmagic", "#### LoaderInfo: systemd-stub " GIT_VERSION
 
 DECLARE_SBAT(SBAT_STUB_SECTION_TEXT);
 
+#define ADDON_FILENAME_EVENT_TAG_ID 0x6c46f751
```

Comment:
please enclose in UINT32_C()

where does the id come from? i hope /dev/urandom?

Reply:
openssl rand -hex 16